Troubleshooting/README.md

```diff
@@ -180,134 +178,185 @@ public class HelloFunction {
-    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
-    at java.base/java.lang.reflect.Method.invoke(Method.java:564)
-
-ERROR: exit status 1
+Deploying trouble to app: tutorial
+Bumped to version 0.0.2
+Building image trouble:0.0.2
+Updating function trouble using image trouble:0.0.2...
+Successfully created app:  tutorial
+Successfully created function: trouble with trouble:0.0.2
 ```
 
-But things are different when you deploy your function to a (likely) remote Fn
-server.  When you do that we have to use the logs to see runtime errors. 
+You can verified the function is deployed successfully by listing
+the functions of the 'tutorial' app:
 
-Deploy the function to an Fn server.  If you haven't got a server running
-locally you can follow the [Install and Start Fn](../install/README.md) tutorial
-to get setup.
+![user input](images/userinput.png)
+>```sh
+> fn ls functions tutorial
+```
+
+Or the slightly more economical:
 
 ![user input](images/userinput.png)
 >```sh
-> fn deploy --app tutorials --local
->```
+> fn ls f tutorial
+```
 
 ```sh
-Deploying trouble to app: tutorials at path: /trouble
-Bumped to version 0.0.2
-Building image trouble:0.0.2
-Updating route /trouble using image trouble:0.0.2...
+NAME     IMAGE          ID
+trouble  trouble:0.0.1  01CT1QZFJTNG8G00GZJ0000002
 ```
 
-If we call the deployed function we get an error message that the call failed,
-but no details.
+With the function defined let's invoke it and see what happens when if fails:
 
 ![user input](images/userinput.png)
 >```sh
-> fn call tutorials /trouble
+> fn invoke tutorial trouble
 >```
 
 ```sh
-ERROR: error calling function: status 500
+Fn: Error calling function: status 502
+
+See 'fn <command> --help' for more information. Client version: 0.5.16
 ```
 
-We need to get the log for the failed call to see what happened.  The first
-step is to get a list of calls for the `tutorials` application.  You can do
-this with the `l` or `list` command. The syntax is `fn [l|list] calls app-name`.  
-Let's get the calls for our `tutorials` app:
+This is not much information to go on to debug the problem.  What we need to
+do is look at the logs!
+
+### Log Capture
+
+We need to capture the logs for the function so that we can see what happens
+when it fails.  To capture logs you need to configure the `tutoral` application
+with the URL of a syslog server.  You can do this either when you create an
+app or after it's been created.  
+
+When creating a new app you can specify the URL using the `--syslog-url` option
+as in:
+
+```sh
+fn create app tutorial --syslog-url tcp://mysyslogserver.com
+```
+
+Since we've already created the 'tutorial' app, we'll have to update it using
+`fn update app`.  But before we do that we'll need
+a syslog server ready to receive log data.  For the purposes of this
+tutorial we'll setup and use a free [Papertrail](https://papertrailapp.com/)
```

Comment:
Since Fn supports a syslog sink, one can also use ELK stack thanks to the Logstash syslog input plugin which can accept the logs, forward them to Elasticsearch and all the way to Kibana - this can be mentioned here as well as a FYI

Reply:
Can you give me a suggested edit to the README?